package OTDB;

message BitcoinAcct_InternalPB {
  optional string bitcoin_id = 1;
  optional string bitcoin_name = 2;
  optional string gui_label = 3;
}

message Wallet_InternalPB {
  repeated BitcoinAcct_InternalPB bitcoin_acct = 1;
}

